addon/components/frost-info-bar.js

```diff
@@ -5,5 +5,6 @@ export default Component.extend({
   // == Properties ============================================================
 
   classNames: ['frost-info-bar'],
-  layout
+  layout,
+  hook: 'info-bar'
```

Comment:
If we wanna talk about "should" it should never have had a default to begin with. Depends on if we want to make a major version bump when do  "remove spacebar heating" bug fixes or not. Personally, I have no problem making breaking changes, and major bumps, even for bugfixes, even if we didn't or shouldn't have supported the previous buggy implementation, but I'm not sure we have ever agreed upon that. Any bugfix that changes things would then be breaking, if someone was dependent on it and using it the broken way. Like that CSS fix for Firefox, I might have set styles in my app that depended on that bug, and changing it would be breaking, right?